doc/releases/migration-guide-3.7.rst

```diff
@@ -39,6 +39,12 @@ Boards
 Modules
 *******
 
+MbedTLS
+=======
+
+* The default enabling of the hash algorithms SHA-384, SHA-512, MD5 and SHA-1 has been removed.
```

Comment:
Just a nit , the grammar is a bit funny.
```suggestion
* Hash algorithms SHA-384, SHA-512, MD5 and SHA-1 are not enabled by default anymore.
```

Reply:
Haha I had it this way but changed it last second, seems like I shouldn't have.